.github/workflows/Build.yml

```diff
@@ -45,86 +46,125 @@ jobs:
-        for region in "${regions[@]}"
-        do
-           for policy in "${policies[@]}"
-           do
-               sudo podman run --rm --name cloud-governance -e policy=$policy -e AWS_ACCESS_KEY_ID=$AWS_ACCESS_KEY_ID -e AWS_SECRET_ACCESS_KEY=$AWS_SECRET_ACCESS_KEY -e AWS_DEFAULT_REGION=$region -e dry_run=yes -e policy_output=${{ secrets.OUTPUT_BUCKET }}/test/$region -e log_level=INFO quay.io/ebattat/cloud-governance
-           done
-        done
-        
-        # gileaks policy
-        # GITHUB_TOKEN already taken => GIT_TOKEN
-        region='us-east-1'
-        policy='gitleaks' 
-        sudo podman run --rm --name cloud-governance -e policy=$policy -e AWS_ACCESS_KEY_ID=$AWS_ACCESS_KEY_ID -e AWS_SECRET_ACCESS_KEY=$AWS_SECRET_ACCESS_KEY -e AWS_DEFAULT_REGION=$region -e git_access_token=${{ secrets.GIT_TOKEN }} -e git_repo=https://github.com/redhat-performance -e several_repos=yes -e policy_output=${{ secrets.OUTPUT_BUCKET }}/test/$region -e log_level=INFO quay.io/ebattat/cloud-governance
-    - name: 🎁 Bump Version
-      run: |
-        version=$(python3 setup.py --version)
-        git checkout main
-        pip install bumpversion
-        git config --global user.email ${{ secrets.EMAIL }}
-        git config --global user.name  ${{ secrets.NAME }}
-        git config pull.rebase false  # merge (the default strategy)
-        bumpversion patch
-        # GITHUB_REPOSITORY already taken => GIT_REPOSITORY
-        git commit .bumpversion.cfg setup.py -m 'bump version to exist version v'$version
-        git pull https://${{ secrets.GITHUB_TOKEN }}@${{ secrets.GIT_REPOSITORY}} main
-        git push https://${{ secrets.GITHUB_TOKEN }}@${{ secrets.GIT_REPOSITORY}} main
-        git push https://${{ secrets.GITHUB_TOKEN }}@${{ secrets.GIT_REPOSITORY}} --tag
+          done
+      - name: ⌛ Upload to 🐋 quay.io
+        run: |
+          version=$(python3 setup.py --version)
+          sudo podman build --build-arg VERSION=$version --layers=false -t ${{ secrets.PACKAGE_NAME }}:v$version .
+          sudo podman login quay.io -u ${{ secrets.QAUYIO_ROBOT_USER }} -p ${{ secrets.QAUYIO_ROBOT_TOKEN }}
+          sudo podman push localhost/${{ secrets.PACKAGE_NAME }}:v$version quay.io/${{ secrets.QAUYIO_REPOSITORY }}/${{ secrets.PACKAGE_NAME }}:v$version
+          sudo podman push localhost/${{ secrets.PACKAGE_NAME }}:v$version quay.io/${{ secrets.QAUYIO_REPOSITORY }}/${{ secrets.PACKAGE_NAME }}
+          echo 'Wait 30 sec till image will be updated in quay.io'
+          sleep 30
+
+  e2e:
+
+      name: e2e
+      runs-on: ubuntu-latest
+      needs: [ test, build ]
+
+      steps:
+        - uses: actions/checkout@v2
+        - name: Set up Python 3.9
+          uses: actions/setup-python@v2
+          with:
+            python-version: 3.9
+      - name: ✔️ E2E tests using latest quay.io
+        run: |
+          declare -a regions=('us-east-1' 'us-east-2' 'us-west-1' 'us-west-2' 'eu-central-1')
```

Comment:
Add matrix strategy for region and policy

Reply:
ok sir.